packages/react-components/react-search/src/components/SearchBox/useSearchBoxStyles.styles.ts

```diff
@@ -62,6 +54,11 @@ const useContentAfterStyles = makeStyles({
     paddingLeft: tokens.spacingHorizontalM,
     columnGap: tokens.spacingHorizontalXS,
   },
+  rest: {
+    opacity: 0,
+    height: 0,
+    weight: 0,
```

Comment:
Guessing this is a typo :D
```suggestion
    width: 0,
```